mmpose/datasets/datasets/base/kpt_3d_mview_rgb_img_direct_dataset.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from abc import ABCMeta, abstractmethod
+
+import json_tricks as json
+import numpy as np
+from torch.utils.data import Dataset
+
+from mmpose.datasets import DatasetInfo
+from mmpose.datasets.pipelines import Compose
+
+
+class Kpt3dMviewRgbImgDirectDataset(Dataset, metaclass=ABCMeta):
```

Comment:
What does `direct` mean here?

Reply:
This means directly estimating 3D human poses instead of obtaining 2D poses firstly.